ml3d/vis/visualizer.py

```diff
@@ -1496,7 +1496,8 @@ def visualize_dataset(self,
         # Setup the labels
         lut = LabelLUT()
         for val in sorted(dataset.label_to_names.keys()):
```

Comment:
you could replace dataset.label_to_names.keys() by dataset.label_to_names.values() and remove class_name = dataset.label_to_names[val]

Reply:
Done.